i18n/es.json

```diff
@@ -3203,6 +3203,9 @@
   "help.messaging.reply": "**Responde a los mensajes** haciendo clic en la flecha que aparece junto al mensaje de texto.",
   "help.messaging.title": "Mensajería Básica",
   "help.messaging.write": "**Escribe mensajes** usando la caja de entrada de texto en la parte inferior de Mattermost. Presiona RETORNO para enviar un mensaje. Usa MAYUS+RETORNO para crear una nueva línea sin enviar el mensaje.",
+  "imageToolbarZoomDropdown.automatic": "Automático",
```

Comment:
please only change the `en.json` file for translations ... all others are automatically generated from weblate